tsdb/functions.go

```diff
@@ -432,9 +435,9 @@ func MapMean(input *MapInput) interface{} {
 		out.Count++
 		switch v := item.Value.(type) {
 		case float64:
-			out.Mean += (v - out.Mean) / float64(out.Count)
+			out.Total += v
```

Comment:
The tests were non-deterministic because of the excessive float math we were doing. This limits us from doing insanely large float math but is that an issue?